sdk/helper/jsonutil/json.go

```diff
@@ -16,12 +16,8 @@ func EncodeJSON(in interface{}) ([]byte, error) {
 	if in == nil {
 		return nil, fmt.Errorf("input for encoding is nil")
 	}
-	var buf bytes.Buffer
-	enc := json.NewEncoder(&buf)
-	if err := enc.Encode(in); err != nil {
-		return nil, err
-	}
-	return buf.Bytes(), nil
+
```

Comment:
`jsonutil` is used _all over_. I'd be concerned that this is going to affect other parts of Vault if it's changing how marshaling is happening. Notably I'm not sure why it's using Encode in the first place instead of Marshal, but my guess is to keep it minimal, that is, to _not_ add newlines, in which case this change would fundamentally alter the use-case of this library.